### DISCUSSION TITLE
How to check is "system" is "light" or "dark"?

### DISCUSSION THREAD

**Discussion by ghost** [TomSchimansky/CustomTkinter#1530] [Q&A] [answered]
I'm making a [project](https://github.com/dishb/hive) with `customtkinter`. When I set the appearance mode to "system" with,
```python
 customtkinter.set_appearance_mode("system")
```
, I want to know if the system appearance mode is "light" or "dark". Any suggestions on how to do this?

**Comment by Akascape** [comment, marked as answer]
Use `customtkinter.get_appearance_mode()`

**Reply by ghost** [reply]
@Akascape - Thank you!
